bpm/src/main/java/org/jboss/pnc/bpm/BpmManager.java

```diff
@@ -64,7 +64,7 @@
     private static final int AUTHENTICATION_TIMEOUT_S = 2 * 60;
 
     private Configuration configuration;
-    private BpmModuleConfig bpmConfig;
+    protected BpmModuleConfig bpmConfig;
```

Comment:
Does it need to be protected ? Looks like something is not right, If it is for test purpose only.